gabrielaEnache/src/main/java/collections/store/Item.java

```diff
@@ -0,0 +1,44 @@
+package collections.store;
+
+import static collections.utils.Generator.generateId;
+
+public class Item {
+    public long id;
```

Comment:
The same, if you are using getters and setters, then you can modify the access modifier to private

Reply:
Fixed